cypress/integration/menu/position.feature

```diff
@@ -1,42 +1,46 @@
 Feature: Position of a menu component
 
+    # default max width: 380px; default max height: 380px
     Background:
-        # the default maximum
-        Given the menu component has a height of 368px
-        # also the default maximum
-        And the menu component has a width of 380px
-        And the menu component is triggered by a user click on an anchor element
+        Given the menu component has a height and height of its default maximum
 
     Scenario: Default rendering
-        Given the anchor has more than 368px space below, before the end of the <body> element
-        When the menu is rendered
-        Then the menu is rendered below the anchor
+        Given there is enough space below the anchor to fit the default maximum
+        When the menu is opened
+        Then the menu is below the anchor
         And the menu is left aligned with the anchor
```

Comment:
Should we be more precise here? `The left of the menu is aligned with the left side of the anchor`. Seems more clear to me.